WORKSPACE

```diff
@@ -0,0 +1,23 @@
+workspace(name = "autocxx.rs")
```

Comment:
`WORKSPACE` is deprecated and disabled by default in the most recent Bazel version.

I would use a `MODULE.bazel` file if possible.